diff --git a/extras/AFC_lane.py b/extras/AFC_lane.py
index 38fd275..06394ea 100644
--- a/extras/AFC_lane.py
+++ b/extras/AFC_lane.py
@@ -652,6 +652,7 @@ class AFCLane:
                     self.afc.spool.clear_values(self)
                     self.afc.function.afc_led(self.afc.led_not_ready, self.led_index)
 
+        self.fila_load.runout_helper.min_event_systime = self.reactor.monotonic() + self.fila_load.runout_helper.event_delay
         self.afc.save_vars()
 
     def prep_callback(self, eventtime, state):
@@ -766,6 +767,7 @@ class AFCLane:
                 self.afc.spool.clear_values(self)
                 self.unit_obj.lane_unloaded(self)
 
+        self.fila_prep.runout_helper.min_event_systime = self.reactor.monotonic() + self.fila_prep.runout_helper.event_delay
         self.afc.save_vars()
 
     def do_enable(self, enable):
